advice/chris-e-advice.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="./css/advice-chris.css">
```

Comment:
We'll need this to be `href="../css/advice-chris.css"` (double dots at the beginning of the path)